app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaScreenModel.kt

```diff
@@ -718,6 +721,23 @@ class MangaScreenModel(
                 read = read,
                 chapters = chapters.toTypedArray(),
             )
+
+            if (read) {
```

Comment:
Should only show the snackbar if max chapter number is less than what is in track